src/test/resources/com/washingtonpost/arc/ans/v0_3/model/text-fixture-good-channels.json

```diff
@@ -0,0 +1,6 @@
+{
+    "_id": "0987654321",
```

Comment:
otherwise, as long as this "mvn clean install"s this looks good to merge; fire away